src/io/flutter/run/LaunchState.java

```diff
@@ -81,6 +85,16 @@ public LaunchState(@NotNull ExecutionEnvironment env,
     DaemonConsoleView.install(this, env, workDir);
   }
 
+
+  @Nullable
+  protected ConsoleView createConsole(@NotNull final Executor executor) throws ExecutionException {
+    if (FlutterLog.LOGGING_ENABLED) {
+      return new FlutterLogView();
+    }
+
```

Comment:
perhaps add an `else` to clarify its one or the other?

Reply:
Done.